setup.py

```diff
@@ -60,6 +60,7 @@ def read_long_description():
         "requests",
         "six",
         "tabulate",
+        "pyyaml==5.2",  # pyYaml stopped supporting python 3.4 after this
```

Comment:
We've been dropping Python 3.4 support from other libraries since it's reached end-of-life (and unlike 2.7 we have no other practical need to support it). Could we relax this version constraint by just dropping Python 3.4 support?

Reply:
Cool. I'll open another PR to drop Python 3.4. I'll do this as separate PR independently of the PyYaml dependancy

Reply:
Dealt with here: https://github.com/facultyai/faculty-cli/pull/40